src/libs/getPlaidLinkTokenParameters/index.ios.js

```diff
@@ -0,0 +1,3 @@
+import CONFIG from '../../CONFIG';
+
+export default () => ({redirect_uri: `${CONFIG.EXPENSIFY.URL_EXPENSIFY_CASH}partners/plaid/oauth_ios`});
```

Comment:
should we add a staging URL here or assume we're just going to test on prod?

Reply:
Ah hmm that's a good question. This url will be whatever we set it to here...

https://github.com/Expensify/App/blob/e24b86b5217ef885f6da0eb1b17bbcfec143d13d/src/CONFIG.js#L10
https://github.com/Expensify/App/blob/e24b86b5217ef885f6da0eb1b17bbcfec143d13d/.env.staging#L1
https://github.com/Expensify/App/blob/e24b86b5217ef885f6da0eb1b17bbcfec143d13d/.env.production#L1

But my understanding is that it will only be used to redirect back to the app and not actually navigate us anywhere. And in that case the iOS app has universal links set up for both staging and production - see here:

https://github.com/Expensify/App/blob/f31be6f12404bb8ed7aecb0115d422e720a0a297/ios/NewExpensify/Chat.entitlements#L9-L13

Which means that a link to either endpoint will always be handled as a universal link by the app. So basically, my guess is that it doesn't matter at all. LMK if that makes sense.